pep-0240.txt

```diff
@@ -2,94 +2,108 @@ PEP: 240
+have been addressed to some extent by the acceptance of PEP 327
+for decimal arithmetic.  Guido also noted, "Rational arithmetic
+was the default 'exact' arithmetic in ABC and it did not work out as
+expected".  See the python-dev discussion on 17 June 2005 [2]_.
 
-    This PEP is rejected.  The needs outlined in the rationale section
-    have been addressed to some extent by the acceptance of PEP 327
-    for decimal arithmetic.  Guido also noted, "Rational arithmetic
-    was the default 'exact' arithmetic in ABC and it did not work out as
-    expected".  See the python-dev discussion on 17 June 2005.
 
 Rationale
+=========
 
-    Rational numbers are useful for exact and unsurprising arithmetic.
-    They give the correct results people have been taught in various
-    math classes.  Making the "obvious" non-integer type one with more
-    predictable semantics will surprise new programmers less than
-    using floating point numbers. As quite a few posts on c.l.py and
-    on tutor@python.org have shown, people often get bit by strange
-    semantics of floating point numbers: for example, round(0.98, 2)
-    still gives 0.97999999999999998.
+Rational numbers are useful for exact and unsurprising arithmetic.
+They give the correct results people have been taught in various
+math classes.  Making the "obvious" non-integer type one with more
+predictable semantics will surprise new programmers less than
+using floating point numbers. As quite a few posts on c.l.py and
+on tutor@python.org have shown, people often get bit by strange
+semantics of floating point numbers: for example, round(0.98, 2)
+still gives 0.97999999999999998.
 
 
 Proposal
+========
 
-    Literals conforming to the regular expression '\d*.\d*' will be
-    rational numbers.
+Literals conforming to the regular expression '\d*.\d*' will be
+rational numbers.
 
 
 Backwards Compatibility
+=======================
 
-    The only backwards compatible issue is the type of literals
-    mentioned above.  The following migration is suggested:
+The only backwards compatible issue is the type of literals
+mentioned above.  The following migration is suggested:
 
-    1. The next Python after approval will allow 
-       "from __future__ import rational_literals" 
-       to cause all such literals to be treated as rational numbers.
+1. The next Python after approval will allow
+   "from __future__ import rational_literals"
+   to cause all such literals to be treated as rational numbers.
 
-    2. Python 3.0 will have a warning, turned on by default, about
-       such literals in the absence of a __future__ statement.  The
-       warning message will contain information about the __future__
-       statement, and indicate that to get floating point literals,
-       they should be suffixed with "e0".
+2. Python 3.0 will have a warning, turned on by default, about
+   such literals in the absence of a __future__ statement.  The
```

Comment:
It's rendered as dunder future dunder. But I can change this to ``__future__``  :)

<img width="811" alt="screen shot 2017-01-08 at 9 52 39 am" src="https://cloud.githubusercontent.com/assets/5844587/21752164/aa5c9874-d588-11e6-9c80-89de775a95ae.png">